routes/login/login.controller.js

```diff
@@ -1,27 +1,63 @@
-module.exports = function(app) {
+module.exports = function (app) {
   // redirect from "/login" → "/login/accessCode"
   app.get('/login', (req, res) => res.redirect('/login/code'))
   app.get('/login/code', (req, res) =>
     res.render('login/code', { title: 'Enter access code', data: req.session || {} }),
   )
-  app.post('/login/code', postCode)
+  app.post('/login/code', processCode)
```

Comment:
What do you think of `postLoginCode`, so that we keep the HTTP method name?

Reply:
love it